src/Apps/ArtQuiz/ArtQuizContext/ArtQuizContext.tsx

```diff
@@ -0,0 +1,169 @@
+import {
```

Comment:
No need to create additional folders for just one item. ArtQuizContext can live right in `src/Apps/ArtQuiz`